1241/en.md

```diff
@@ -0,0 +1,51 @@
+# LOJ 1389 - Scarecrow
+
+### Summary:
+
+First, you will be given an integer __T (1 ≤ T ≤ 100)__ denoting the number of test cases. Each case will start with a line containing an integer __n (1 ≤ n ≤ 10)__, where __n__ denotes the total number of reports of the size of Pinocchio's nose in a certain day. The next line contains __n__ space separated integers: __a<sub>1</sub>, a<sub>2</sub>, ..., a<sub>n</sub>__ denoting the sizes of his nose in that day. All the values of __a<sub>i</sub> (1 ≤ i ≤ n)__ will be between 2 and 50.
+
+### Idea:
+
+ We have to find the minimum number of lies Pinocchio has told. When he tells a lie, his nose grows at least 1 cm and at most 5 cm. So assuming his nose grows 5 cm for most of the lies he tells will minimize our answer. For example, if his nose grows 13 cm in a certain period of time, then the minimum number of lies will be 3 (we have to work with the ceiling values here).
+
+ As it is assumed that he hasn't told any truth, the given values of __a<sub>i</sub> (1 ≤ i ≤ n)__ will be in non-decreasing order. So we don't have to sort the values. We will use a counter variable to store the answer. We will run a loop __n__ times and work with __a<sub>i</sub>__ and __a<sub>i-1</sub> (1 ≤ i ≤ n)__ in each iteration. Every time we will add the ceiling value of __(a<sub>i</sub> - a<sub>i-1</sub>)/5__ with the counter (Note: In first iteration we have to work with __a<sub>1</sub>__ and the initial size of Pinocchio's nose. So we can assume that __a<sub>0</sub>__ = 2). Finally, we have to print the value of the counter variable. The space and time complexity both will be __O(n)__ for each of the __T__ tast cases.
```

Comment:
This is rigorous implementation details and seems unnecessary. Instead of this, you can write something like 

> From the `i`-th timestamp to the `i+1`-th, Pinocchio has told `ceil((a[i+1] - a[i]) / 5)` lies. And we will simply add this value for each `i` and get our result.

You can also add the consideration for `a[0] = 2` here.